extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,54 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live story
+
+Use the `live-story` attribute to append new pages to a story for users to see in real-time.
+
+This attribute will show a notification of new pages and update the progress bar to users on the last page.
+
+Core use cases for live-story include coverage for breaking news or live events, enabling real-time updates to users without exiting the story. Award shows, sporting events, and elections are some examples.
+
+#### How it works
+
+In the background, while an AMP Story using `live-story` is displayed on the client, the AMP runtime polls the origin document on the host for updates. When the client receives a response, it then filters and dynamically inserts those updates back into the story on the client. Publishers can customize the polling rate in order to control the number of incoming requests, and AMP caches like the Google AMP Cache can perform optimizations to reduce the server response payload, saving client bandwidth and CPU cycles.
+
+#### Polling
+
+In most implementations for live blogs, content is either pushed by the server to the client instance of a page, or the client polls a JSON endpoint to receive updates. The implementation here is different, in that the client instance of the story polls the server copy of the story document for updates inside the `<amp-story>` element. For instance: if the user is viewing a story served from an AMP cache, the client will poll that document hosted on that AMP cache for updates; if the user is viewing a document served from a web publisher's origin domain (e.g. "example.com"), then the client will poll the document hosted on that origin domain for updates.
+
+This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work.
+
+New content just needs to be published to the same URL with a valid amp-story markup. The user will then have that content pulled into their client instance during the next poll (poll intervals are configurable in the component, and are valid above the minimum of 15000 milliseconds).
+
+#### Stop polling
+
+As long as the `live-story` attribute is present on the `<amp-story>` element, the client will make continuous polls to the server copy of the document. Make sure to set the `live-story-disabled` attribute to make the polling stop once you're finished with the live broadcast of your story.
```

Comment:
I would add something here that specifies you add the `live-story-disabled` to the markup when you publish your last update to the story. (if that's how it works?)